docs/.sphinx/_static/css/crossref.css

```
@@ -0,0 +1,9 @@
+.crossref {
+  display: block;
+  font-size: 0.85em;
```

It's a tiny difference, but how about this for consistency with the font size used for the TOC:

```suggestion
  font-size: var(--font-size--small);
```